phpBB/styles/prosilver/template/overall_footer.html

```diff
@@ -11,6 +11,7 @@
 		{CREDIT_LINE}
 		<!-- IF TRANSLATION_INFO --><br />{TRANSLATION_INFO}<!-- ENDIF -->
 		<!-- EVENT overall_footer_copyright_append -->
+		<br><a href="{{ U_PRIVACY }}" title="{{ lang('PRIVACY_LINK') }}" role="menuitem"><span>{{ lang('PRIVACY_LINK') }}</span></a>&nbsp;&vert;&nbsp;<a href="{{ U_TERMS_USE }}" title="{{ lang('TERMS_LINK') }}" role="menuitem"><span>{{ lang('TERMS_LINK') }}</span></a>
```

Comment:
personally I would rather rework/like to see something like this

```html
<div class="copyright">
    <p class="footer-row">
		<span class="footer-copyright">Powered by <a class="footer-brand" href="https://www.phpbb.com/">phpBB</a>® Forum Software © phpBB Limited</span>
	</p><p class="footer-row">
	<a class="footer-link" href="{{ U_PRIVACY }}" title="{{ lang('PRIVACY_LINK') }}" role="menuitem">
		<span class="footer-link-text">Privacy</span>
	</a>
    |
	<a class="footer-link" href="{{ U_TERMS_USE }}" title="{{ lang('TERMS_LINK') }}" role="menuitem">
		<span class="footer-link-text">Terms</span>
	</a>
</p>
	<p class="footer-row">
        <span class="footer-info" title="SQL time: 0.022s / PHP time: 0.108s">Time: 0.130s</span> | <span class="footer-info" title="Cached: 2">Queries: 13</span> | <span class="footer-info">Peak Memory Usage: 3.7 MiB</span>
    </p>
</div>
```

```css
.copyright {
     font-size: 10px;
    text-align: center;
    padding: 10px;
}

.footer-row {
    font-size: 10px;
    line-height: 1.8;
    margin: 0;
}
```

<img width="1230" alt="screenshot 2018-05-31 17 59 37" src="https://user-images.githubusercontent.com/73081/40810734-bdb77fb0-64fc-11e8-826c-fd1df4f82661.png">

Reply:
this gives far more control/theme custimizability

Reply:
if not then at least recommend remove html chars these are no longer required...and just looks weird